Experiments_toolkit.py

```diff
@@ -1,4 +1,4 @@
-import windowing as w
+import windowing as wl
```

Comment:
IMO this file should be separated in plotting utility, augmentation, data preparation